src/lib/ngsi/AttributeList.cpp

```diff
@@ -41,18 +41,19 @@
 */
 std::string AttributeList::render(Format format, const std::string& indent, bool comma)
```

Comment:
I see several changes in some renders... why are the needed? Just to understand it...